Anguandia/quickcredit
# Add signup functionality

Accomplish basic signup controller functionality with duplicate
validation and activate successful signup tests

To test this feature
- Clone this repo and install with npm install
- Run tests with mocha tests command; two tests should run and pass
- On postman, enter the URL http://localhost:3000/auth/signup and a raw JSON body with values for first&last names, password, and email at a minimum select the post method and hit the send button; the user should be created
- Resubmit the same request or at least maintain the email and see the response that the user exists

Limitations
- Basic signup and duplicate signup validation only
- Detailed validation to follow
- To be deployed to a hosting platform

[finishes #165845533]
